### PR TITLE
forcing all non-multi geometries to be multi

### PR DESCRIPTION
**Pull request by chelm**
This just wraps non-multis into multi. I actually need to create multi-geom test case though, so give me a little bit and this PR should have a new test with multis just to make sure its all safe and sound.

### CODE REVIEW

**Comment by chelm** [comment]
Yeah hold off need to fix the multi case.

**Comment by ngoldman** [comment]
@chelm looks good to me, merging unless you still want me to hold off.

**Comment by chelm** [comment]
@ngoldman rock n roll

File: test/geojson-multi-spec-example.json [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+      },
+      "properties": {
+        "prop0": "value0",
+        "prop1": 0.0
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Polygon",
```
Comment (ngoldman) [inline test/geojson-multi-spec-example.json]: @chelm sorry just caught this, should this be a `MultiPolygon` type?

Comment (chelm) [inline test/geojson-multi-spec-example.json]: whoops, yes, my bad.

Comment (chelm) [inline test/geojson-multi-spec-example.json]: ...and that breaks the tests, shoot.

**Pull request merged**